WordPress/Classes/Services/BloggingPrompts/BloggingPromptsService.swift

```diff
@@ -260,6 +281,64 @@ private extension BloggingPromptsService {
         return Self.utcDateFormatter.date(from: dateString)
     }
 
+    // MARK: Prompts
+
+    /// Fetches a number of blogging prompts for the specified site from the v3 endpoint.
+    ///
+    /// - Parameters:
+    ///   - number: The number of prompts to query. When not specified, this will default to remote implementation.
+    ///   - fromDate: When specified, this will fetch prompts from the given date. When not specified, this will default to remote implementation.
+    ///   - ignoresYear: When set to true, this will convert the date to a custom format that ignores the year part. Defaults to true.
+    ///   - forceYear: Forces the year value on the prompt's date to the specified value. Defaults to the current year.
+    ///   - completion: A closure that will be called when the fetch request completes.
+    func fetchRemotePrompts(number: Int? = nil,
+                            fromDate: Date? = nil,
+                            ignoresYear: Bool = true,
+                            forceYear: Int? = nil,
+                            completion: @escaping (Result<[BloggingPromptRemoteObject], Error>) -> Void) {
+        let path = "wpcom/v3/sites/\(siteID)/blogging-prompts"
+        let requestParameter: [String: AnyHashable] = {
+            var params = [String: AnyHashable]()
+
+            if let number, number > 0 {
+                params["per_page"] = number
+            }
+
+            if let fromDate {
+                // convert to yyyy-MM-dd format in UTC timezone so users would get the same prompts everywhere.
+                var dateString = Self.utcDateFormatter.string(from: fromDate)
```

Comment:
Is using UTC here correct? I would think that this would cause the daily prompt to change for users at a random point in their day, depending on their timezone.

I checked our current implementation and it uses the [local time](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/ae2d7a1afed338adc03adc013b4a3b223fc1d25d/WordPressKit/BloggingPromptsServiceRemote.swift#L40).

Reply:
@wargcm 

Yesterday, I thought our implementation with local time _was_ the faulty one... Because with UTC, everyone would see the same prompts everywhere regardless of location, which would maximize the number of answers submitted that day. 

Plus, I also thought about the web and _assumed_ that it also works this way, considering that the server, which runs in UTC, determines what prompts the user sees at the moment. And since we are switching to v3 anyway, I'd thought to change it to UTC, too.

> this would cause the daily prompt to change for users at a random point in their day, depending on their timezone.

But yeah... Now that I think about it, this makes more sense. Plus, my assumption about the web is wrong 😆. It looks like I'm getting the prompts based on my local time, despite the web having less emphasis on _today's_ prompt — unlike the apps.

So yeah, I've reverted the implementation in https://github.com/wordpress-mobile/WordPress-iOS/pull/21952/commits/8258b09f170f4a1be7fe990f98835ccc17d907f1. Can you take a look at this again, please? Thanks!